packages/styles/scss/components/ui-shell/side-nav/_side-nav.scss

```diff
@@ -68,7 +68,7 @@
     width: 0;
   }
 
-  .#{$prefix}--side-nav.#{$prefix}--side-nav--rail:not(
+  .#{$prefix}--side-nav.#{$prefix}--side-nav--rail.#{prefix}--cds--side-nav--expand-on-hover:not(
       .#{$prefix}--side-nav--fixed
     ):hover,
```

Comment:
```suggestion
```

It seems like we can just remove this block, and just leave `.#{$prefix}--side-nav--expanded`